cluster/addons/prometheus/README.md

```diff
@@ -0,0 +1,71 @@
+# Prometheus in Kubernetes
+
+This is an experimental [Prometheus](http://prometheus.io/) setup for monitoring
+Kubernetes services that expose prometheus-friendly metrics through address
+http://service_address:service_port/metrics.
+
+# Purpose
+The purpose of the setup is to gather performance-related metrics during load
```

Comment:
Its a little confusing that this is a cluster addon, for this to be useful doesn't it have to be a standalone thing that multiple load tests can send their metrics to across a week? how do other e2e clusters discover the dashboard?